xml/System/Uri.xml

```diff
@@ -71,7 +71,7 @@ var uri = new Uri("http://myUrl/%2E%2E/%2E%2E"); // http scheme, escaped
 OR
 var uri = new Uri("tcp://myUrl/../.."); // ftp scheme, unescaped
 OR
-var uri = new Uri("tcp://myUrl/%2E%2E/%2E%2E"); // ftp scheme, unescaped
+var uri = new Uri("tcp://myUrl/%2E%2E/%2E%2E"); // ftp scheme, escaped
```

Comment:
The comment talks about the "ftp scheme", but the string contains `tcp://`. Shouldn't it be changed to `ftp://` (both here and the line above)?

Reply:
And I missed that this was already merged, will submit a PR and delete the comments.